Setup.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+# EXPERIMENTAL!
+# This file is experimental and is not maintained directly by Improbable. Please use at your own risk.
+
+set -e -u -o pipefail
+if [ -n "${TEAMCITY_CAPTURE_ENV:-}" ]; then set -x; else set +x; fi
+
```

Comment:
Is this meant to only run on Mac? If so, please add:
```suggestion

if [ "$(uname -s)" != "Darwin" ]; then
    echo "This script should only be used on a Mac. If you are using Windows, please run Setup.bat."
    exit 1
fi

```